__tests__/utils/get-base-options.spec.js

```diff
@@ -25,6 +25,20 @@ describe('getBaseOptions', () => {
     getBaseOptions();
     expect(prompts).toHaveBeenCalledTimes(1);
     // snapshot params as its a large array that will grow over time.
+    // test prompts validation
+    expect(prompts.mock.calls[0][0][0].validate('testProjectName')).toBe(true);
+    expect(prompts.mock.calls[0][0][0].format('testProjectName')).toBe('testProjectName');
+    expect(prompts.mock.calls[0]).toMatchSnapshot();
+  });
+  it('should call prompts with the correct set of options when regex is passed', () => {
+    const testRegex = /[^a-z-]+/gi;
+    getBaseOptions(testRegex);
+    expect(prompts).toHaveBeenCalledTimes(1);
+    // snapshot params as its a large array that will grow over time.
```

Comment:
I'm not following why this isnt covered by the snapshot

Reply:
this was for branch coverage, I'm not sure why it was not

Reply:
new assertions shouldn't affect the coverage report